pymaker/dss.py

```diff
@@ -696,122 +672,52 @@ def rho(self, ilk: Ilk) -> int:
         return Web3.toInt(self._contract.call().ilks(ilk.toBytes())[1])
 
     def __repr__(self):
-        return f"Drip('{self.address}')"
+        return f"Jug('{self.address}')"
 
 
-class Pit(Contract):
-    """A client for the `Pit` contract.
+class Cat(Contract):
+    """A client for the `Cat` contract, used to liquidate unsafe Urns (CDPs).
+    Specifically, this contract is useful for Flip auctions.
 
-    Ref. <https://github.com/makerdao/dss/blob/master/src/frob.sol>
+    Ref. <https://github.com/makerdao/dss/blob/master/src/bite.sol>
```

Comment:
I think this should be pointing to https://github.com/makerdao/dss/blob/master/src/cat.sol On this note contracts are using LogNote event and I cannot find any LogBite event, can you pls clarify?